### PR TITLE
Support KITTI 3D format

### PR DESCRIPTION
**Pull request by sooahleex**
### Summary

- Ticket no.151424
- Support KITTI 3D format

### How to test

- Add unit test

### Checklist

- [X] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [X] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [ ] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2024 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: src/datumaro/plugins/specs.json [hunk 1]
```diff
@@ -1,13 +1,4 @@
...
-  {
-    "import_path": "datumaro.plugins.accuracy_checker_plugin.ac_launcher.AcLauncher",
-    "plugin_name": "ac",
-    "plugin_type": "Launcher",
-    "extra_deps": [
-      "tensorflow",
-      "openvino.tools.accuracy_checker"
-    ]
-  },
   {
```
Comment (jihyeonyi) [inline src/datumaro/plugins/specs.json]: Is it related to kitti 3d? Why is this removed?

Comment (sooahleex) [reply src/datumaro/plugins/specs.json]: This is not related to kitti3d, I have fixed this in [a9727ff](https://github.com/openvinotoolkit/datumaro/pull/1619/commits/a9727ff8b6155ceab950d0b0bedea0ab400a0d0c). Thanks for bringing this to my attention.

**Review by jihyeonyi — APPROVED** [review]

**Review by wonjuleee — APPROVED** [review]

**Comment by kprokofi** [comment]
@sooahleex  @wonjuleee 
Thanks for providing this dataset format.
I found 1 stopper issue. Currently, this dataset cannot detect subsets. To align it with other datasets, I would ask you to add possibility to detect subfolders like:
image

Currently, if I even iterate over subsets and create Datumaro datasets, each for each subset, it will lead to labels mismatch and it cannot be used, unfortunately:
image
image
As you can see, labels for train and val are with different indexes.

**Comment by kprokofi** [comment]
@sooahleex One more serious problem:
that media (or image) is something weird. 
image
In KITTI datatset image is just a .png file, not PointCloud and the path to that file is also weird. Could you take a look and fix that as well?

image
